examples/pytorch_imagenet_resnet50.py

```diff
@@ -40,6 +41,10 @@
                     help='SGD momentum')
 parser.add_argument('--wd', type=float, default=0.00005,
                     help='weight decay')
+parser.add_argument('--batches-per-allreduce', type=int, default=1,
+                    help='number of backward iteration steps '
```

Comment:
Mayber rephrase to `number of batches processed locally before executing allreduce across workers; it multiplies total batch size.`